packages/mu-http/driver.js

```diff
@@ -14,4 +14,144 @@
 
 'use strict'
 
-// TODO
+var http = require('http')
+var stringify = require('fast-safe-stringify')
+var parse = require('fast-json-parse')
+var assert = require('assert')
+var mue = require('mu-error')()
+
+
+/**
+ * HTTP transport
+ * Figure out path mapping - perhaps this may need to be part of the config for now just /
+ */
+module.exports = function createTcpDriver (options) {
+  return function httpDriver (opts, receive) {
+    var connections = {}
+    var connectionsByIp = {}
+    var server
```

Comment:
server object is returned from `listen` as in tcp transport, can we assign the return value of listen as in tcp transport (see comment further down)